examples/main.go

```diff
@@ -80,3 +177,26 @@ func main() {
 	enabled, _ = app.IsFeatureEnabled("mutext_feat", user)
 	fmt.Printf("Is feature enabled? %v\n", enabled)
 }
+
+var RunMemProfile = "false"
+var RunCPUProfile = "false"
+
+func main() {
+
+	if RunMemProfile == "true" || RunCPUProfile == "true" {
+
+		const RUN_NUMBER = 6
```

Comment:
Why 6? Not very "stressful" :)

Reply:
a minimum number to see what's going on, increasing this does not have any substantial impact on profiling, but I can set it to something bigger